Resources/AzureDevops/azure-pipelines.yaml

```diff
@@ -0,0 +1,83 @@
+# dotnet core build
+
+pool:
+  name: NCS - CI and CD
+
+variables:
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'release'
+  SolutionBaseName: 'Dfc.ProviderPortal.Courses'
+
+resources:
+  repositories:
+  - repository: self
+  - repository: dfc-devops
+    type: github
+    name: SkillsFundingAgency/dfc-devops
+    endpoint: 'GitHub (ESFA)'
+    ref: refs/tags/v1.9.1
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+
+pr:
+  branches:
+    include:
+    - master
+
+steps:
+
+# ARM template
+- template: AzureDevOpsTemplates/Build/StepTemplates/dfc-arm-build.yml@dfc-devops
+  parameters:
+    ArmTemplateRoot: '$(System.DefaultWorkingDirectory)\Resources\ArmTemplates'
+
+# restore
+- task: DotNetCoreCLI@2
```

Comment:
I'd have liked to use the dfc-dotnetcore-build.yml file here, but they use a non-standard project structure (they have src and test directories) and I didn't want to change too much at once